pkgs/applications/misc/syncthingtray/default.nix

```diff
@@ -19,30 +19,24 @@
 , kioPluginSupport ? true
 , plasmoidSupport  ? true
 , systemdSupport ? true
+# It is possible to set via this option the exec path that is found in the
+# generated `~/.config/autostart/syncthingtray.desktop` file. See
+# https://github.com/NixOS/nixpkgs/issues/199596#issuecomment-1310136382 If you
+# use home-manager for example, you might need to change this path.
+, autostartExecPath ? "/run/current-system/sw/bin/syncthingtray"
```

Comment:
Doesn't it support relative path (`syncthingtray`) in the desktop file? I was under the impression that was preferred, because it avoids bitrot and doesn't require hardcoding anything.

Reply:
> Doesn't it support relative path (`syncthingtray`) in the desktop file? I was under the impression that was preferred, because it avoids bitrot and doesn't require hardcoding anything.

I'm working now with upstream on making it automatically detect the path of the `syncthingtray`, in case it is wrapped. However, I still think that using this path - `/run/current-system/...` is better, since it doesn't change when your update your system.